packages/eas-cli/src/credentials/android/actions/SetupGoogleServiceAccountKey.ts

```diff
@@ -0,0 +1,80 @@
+import nullthrows from 'nullthrows';
+
+import {
+  CommonAndroidAppCredentialsFragment,
+  GoogleServiceAccountKeyFragment,
+} from '../../../graphql/generated';
+import Log from '../../../log';
+import { promptAsync } from '../../../prompts';
+import { Context } from '../../context';
+import { MissingCredentialsNonInteractiveError } from '../../errors';
+import { AppLookupParams } from '../api/GraphqlClient';
+import { AssignGoogleServiceAccountKey } from './AssignGoogleServiceAccountKey';
+import { CreateGoogleServiceAccountKey } from './CreateGoogleServiceAccountKey';
+import { UseExistingGoogleServiceAccountKey } from './UseExistingGoogleServiceAccountKey';
+
+export class SetupGoogleServiceAccountKey {
+  constructor(private app: AppLookupParams) {}
+
+  private async isGoogleServiceAccountKeySetupAsync(ctx: Context): Promise<boolean> {
+    const appCredentials = await ctx.android.getAndroidAppCredentialsWithCommonFieldsAsync(
+      this.app
+    );
+    return !!appCredentials?.googleServiceAccountKeyForSubmissions;
+  }
+
+  public async runAsync(ctx: Context): Promise<CommonAndroidAppCredentialsFragment> {
+    if (ctx.nonInteractive) {
+      throw new MissingCredentialsNonInteractiveError(
+        'Google Service Account Keys cannot be setup in --non-interactive mode.'
```

Comment:
```suggestion
        'Google Service Account Keys cannot be set up in non-interactive mode.'
```
Or maybe `setup` is actually a verb? 🤔

Reply:
I have the same question for other log messages in this file :P

Reply:
You are correct. `setup` is a noun and `set up` is a verb :) http://www.enhancemywriting.com/setup-vs-set-up